lib/util.lua

```diff
@@ -1,9 +1,19 @@
 local util = {}
 --TODO: the whole stencil/canvas system should be reviewed since it has been changed in a naive way
 
+local temp = nil
+
 function util.process(canvas, options)
   --TODO: now you cannot draw a canvas to itself  
-  temp = love.graphics.newCanvas()
+  if not temp then
```

Comment:
Maybe this change should be done in PR #28 where discussion is being held on it? Could you take out this commit and either show this code as example in the other PR or open a PR with this change alone?

Reply:
It looks like PR #28 is clean, if we can get it commited and have him remove this it should be good to commit yes?